src/file/file.service.ts

```diff
@@ -106,13 +114,15 @@ export class FileService {
     return file;
   }
 
-  public static async getFilesByFolder(folderID: string | null, ownerID: string | null): Promise<any> {
+  // isDeleted chooses if it would send back the deleted files or not. by default retrieves non-deleted.
```

Comment:
This comment is still not helpful.

Reply:
why? it is not clear from the name. How would you recommend to explain it?

Reply:
This comment only explains what `isDeleted` is used for, but doesn't explain what the function does.

Reply:
So it doesn't mean its not helpful. Its true that functions should have more comments. Deleting helpful comments won't fix it.

Reply:
Just comment the function :angry: